pkg/cloud/tags/tags_test.go

```diff
@@ -20,7 +20,42 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/golang/mock/gomock"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
```

Comment:
To be consistent with the rest of the tests, cluster api ecosystem is using gomega assertions.
cc @shivi28 @Madhur97

Reply:
Done